tests/test_taxonomy_bakta.nf.test

```diff
@@ -0,0 +1,62 @@
+nextflow_pipeline {
+
+    name "Test pipeline: NFCORE_FUNCSCAN"
+    script "main.nf"
+    tag "pipeline"
+    tag "nfcore_funcscan"
+    tag "test_taxonomy_bakta"
+
+    test("test_taxonomy_bakta") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+                run_taxa_classification = true
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+
+                { assert snapshot(
```

Comment:
These should be moved into their own `match` groups like the other tests

Reply:
Done in [33f55b5](https://github.com/nf-core/funcscan/pull/366/commits/33f55b5306573c630c959965bea94a71aef74375).